js/main.js

```diff
@@ -16,7 +16,24 @@ $(document).ready(function () {
     if (logoID < 10) {
         logoID = "0" + logoID;
     }
-    document.querySelector('#defaultIcon1').href = 'https://www.sugarlabs.org/assets/favicon_' + logoID + '.png';
```

Comment:
Code you are removing has nothing to do with the issue; the issue and my tests of master branch just now show the error to be a reference to press/assets/favicon_06.png and this line you are removing is an absolute link without press.

Reply:
I have simplified the favicon logic by making the path relative to `/press/`, removing the full domain URL, and eliminating unnecessary existence checks, as shown in the following code:

```javascript
var defaultIcon = document.querySelector('#defaultIcon1');
if (defaultIcon) {
    var logoID = colorIndex + 1;
    if (logoID < 10) {
        logoID = "0" + logoID;
    }
    defaultIcon.href = '/press/assets/favicon_' + logoID + '.png';
}
```
Also, I have removed the testing and existence checks. Is this approach fine?

Reply:
You've lost me here.  Why would the path be made absolute from /press/?